content/SCALE/SCALEUIReference/SystemSettings/GeneralSettings.md

```diff
@@ -42,9 +42,67 @@ If you do not save the system configuration before resetting it, you may lose da
 
 ## Support
 
-The *Support* window in the Advanced Settings screen displays the systems general hardware and software specs and contains links to the Documentation Hub, TrueNAS Forums, and enterprise licensing information. 
+The *Support* window displays the systems general hardware and software specs and contains links to the [Documentation Hub](https://www.truenas.com/docs/), [TrueNAS Forums](https://www.truenas.com/community/), and offers [TrueNAS Licensing](https://www.ixsystems.com/support/) information. 
 
-There are also buttons that allow users to add an enterprise license or report bugs via a Jira support ticket.
+![GeneralSettingsSCALESupport](/images/SCALE/GeneralSettingsSCALESupport.png "SCALE General Settings Support View")
+
+{{< tabs "Support Options" >}}
+{{< tab "Add License" >}}
```

Comment:
@jervin777 I'm going to rework the article to no longer use tabs. Our current theme doesn't support linking inside tabbed sections and it's causing reader confusion.